third-party/realdds/src/dds-participant.cpp

```diff
@@ -277,7 +277,7 @@ rsutils::string::slice dds_participant::name() const
 
 std::string dds_participant::print( dds_guid const & guid_to_print ) const
 {
-    return realdds::print( guid_to_print, guid() );
+    return realdds::print( guid_to_print, unknown_guid );
```

Comment:
Why is this GUID unknown? Don't we have a participant GUID?

Reply:
`guid()` created an empty, or unknown, guid. I just made it clearer, and easier to reference.

Reply:
Oh, you're asking why we need to pass the unknown guid here?
You're right, this is a mistake, I think I thought it was `dds_guid()`. Fixed.